internal/backends/qt/qt_accessible.rs

```diff
@@ -0,0 +1,699 @@
+            return item_string_property(m_data, QAccessible::Value);
+        }
+
+        void setCurrentValue(const QVariant &value) override {
+            // FIXME: Implement this?
+            Q_UNUSED(value);
+        }
+
+        QVariant maximumValue() const override {
+            return item_string_property(m_data, VALUE_MAXIMUM);
+        }
+
+        QVariant minimumValue() const override {
+            return item_string_property(m_data, VALUE_MINIMUM);
+        }
+
+        QVariant minimumStepSize() const override {
+            return item_string_property(m_data, VALUE_STEP);
+        }
+
+    private:
+        mutable void *m_data = nullptr;
+    };
+
+    // ------------------------------------------------------------------------------
+    // Slint_accessible_window:
+    // ------------------------------------------------------------------------------
+
+    class Slint_accessible_window : public Slint_accessible {
+    public:
+        Slint_accessible_window(QWidget *widget, void *rust_window) :
+            Slint_accessible(widget, QAccessible::Window, QAccessible::queryAccessibleInterface(qApp)),
+            m_rustWindow(rust_window)
+        { }
+
+        ~Slint_accessible_window()
+        {
+            rust!(Slint_accessible_window_dtor [m_rustWindow: *mut c_void as "void*"] {
+                alloc::rc::Weak::from_raw(m_rustWindow as _); // Consume the Weak wo hold in our void*!
+            });
+        }
+
+        void *rustItem() const override {
+            return root_item_for_window(m_rustWindow);
+        }
+
+        QWidget *qwidget() const override {
+            return qobject_cast<QWidget *>(object());
+        }
+
+        QWindow *window() const override {
+            return qobject_cast<QWidget *>(object())->windowHandle();
+        }
+
+        // properties and state
+        QString text(QAccessible::Text t) const override {
+            switch (t) {
+                case QAccessible::Name: return qobject_cast<QWidget*>(object())->windowTitle();
+                default: return QString();
+            }
+        }
+
+        QAccessible::State state() const override {
+            QAccessible::State state;
+            state.active = 1;
+            state.focusable = 1;
+            return state;
+        }
+
+    private:
+        void *m_rustWindow;
+    };
+
+    int Slint_accessible::childCount() const {
+        if (m_children.isEmpty()) {
+            auto descendents = Descendents(rustItem());
+            for (size_t i = 0; i < descendents.count(); ++i) {
+                auto object = new QObject();
+                auto ai = new Slint_accessible_item(descendents.itemAt(i),
+                    object, descendents.roleAt(i),
+                    const_cast<Slint_accessible *>(this));
+                QAccessible::registerAccessibleInterface(ai);
+                m_children.append(object);
+            }
+        }
+        return m_children.count();
```

Comment:
How do we track the changes if the repeater children count change?  Where do you invalidate the tree in that case?

Reply:
We don't. The whole thing is not very dynamic at this time. We *should* watch the dynamic nodes somehow and make them invalidate their subtrees.

How can I actually get notified when the tree changes?

Reply:
> How can I actually get notified when the tree changes?

I'm afraid you can't do that easily.

But you can detect something is out of date by getting the count again and see if any item weak has gone None. and then re-create the tree.

Reply:
I would need to send events when accessible objects get created/deleted/reordered.

Can we maybe trigger something in the window whenever something in the tree of component changes? That would suffice for a11y: We could then create a fresh tree and compare that to the old one.

That would be a pretty expensive operation, but then I do not expect accessibility to be a performance critical operation -- it sends lots of requests over DBus on Linux anyway.

Reply:
I tend to consider this out of scope for this PR:-)

Reply:
> Can we maybe trigger something in the window whenever something in the tree of component changes?

I believe this is the free_graphics_resources call.
We just need to add a call when the component is created. 
Or you can just check in the renderer if you are rendering items for components which are not registered for accessibility